### PR TITLE
🖼 Make storefront illustration responsive

### LINKED ISSUE #1728: Create responsive SVG for the frontpage in Zeroheight
Since we can’t use media queries to control how the content flows in the banner area on the frontpage, we should check if we can use an SVG for the illustration and if so make it responsive to ensure the illustration looks good regardless of viewport width.

### PR DESCRIPTION
**Pull request by vnys**
Resolves #1728

### CODE REVIEW

**Comment by vnys** [comment]
Done with icon positioning, latest version uploaded to https://equinor.zeroheight.com/styleguide/s/74092/p/94d101-eds-official. We now embed a subset of the font, only the letters we actually use. The subset script is included in the repo. It currently expects to find the font in the same directory as the script. This could be streamed directly from the CDN instead, but that will have to wait for a rainy day..

**Review by mimarz — COMMENTED** [review]
Looking good! Some comments and questions.

Also, maybe we should move this under a `assets` folder and then also have the svg files of icons there or what do you think?

File: graphics/woman/woman-with-laptop-icons.svg [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+    <symbol id="Icon-Mic" viewBox="0 0 24 24">
+      <path fill-rule="evenodd" clip-rule="evenodd"
+        d="M15 11.5c0 1.66-1.34 3-3 3s-3-1.34-3-3v-6c0-1.66 1.34-3 3-3s3 1.34 3 3v6zm-3 5c2.76 0 5-2.24 5-5h2c0 3.53-2.61 6.43-6 6.92v3.08h-2v-3.08c-3.39-.49-6-3.39-6-6.92h2c0 2.76 2.24 5 5 5z" />
+    </symbol>
+    <symbol id="Icon-BatteryCharging" viewBox="0 0 24 24">
+      <path fill-rule="evenodd" clip-rule="evenodd"
+        d="M14 4h1.67C16.4 4 17 4.6 17 5.33v15.34c0 .73-.6 1.33-1.34 1.33H8.33C7.6 22 7 21.4 7 20.66V5.33C7 4.6 7.6 4 8.33 4H10V2h4v2zm-3 10.5V20l4-7.5h-2V7l-4 7.5h2z" />
+    </symbol>
+  </defs>
+</svg>
```
[the remark this answers is not in the document]
Comment (vnys) [reply graphics/woman/woman-with-laptop-icons.svg:89]: Yes

File: graphics/font/package.json [hunk 2]
```diff
@@ -0,0 +1,11 @@
+{
+  "scripts": {
+    "start": "node subset.mjs",
+    "serve": "http-server -p 9999"
+  },
+  "dependencies": {
+    "bent": "^7.3.12",
```
[the remark this answers is not in the document]
Comment (vnys) [reply graphics/font/package.json:7]: I don’t think the same module is always the best answer to different needs. Node-fetch is very feature-rich. [Bent](https://github.com/mikeal/bent) is first and foremost minimal, async, has no dependencies and can create a buffer with just a couple of lines of code.

File: graphics/font/package.json [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+  "scripts": {
+    "subset": "node subset.js",
+    "serve": "http-server -p 9999"
+  },
+  "dependencies": {
+    "bent": "^7.3.12",
+    "http-server": "^14.0.0",
+    "subset-font": "^1.4.0"
+  }
+}
```
Comment (vnys) [inline graphics/font/package.json:14]:
```suggestion
}

```

File: graphics/woman/package.json [hunk 4]
```diff
@@ -0,0 +1,10 @@
+{
+  "name": "woman-with-laptop",
+  "version": "1",
+  "scripts": {
+    "dev": "browser-sync start --server '.' --files '*{html,css,svg}' --no-ui --no-open --watch --no-inject-changes"
+  },
+  "dependencies": {
+    "browser-sync": "^2.27.7"
+  }
+}
```
Comment (vnys) [inline graphics/woman/package.json:10]:
```suggestion
}

```

File: graphics/woman/woman-with-laptop-icons.svg (hunk 1 above)
[the remark this answers is not in the document]
Comment (vnys) [reply graphics/woman/woman-with-laptop-icons.svg:89]:
```suggestion
</svg>

```

**Comment by vnys** [comment]
Can somebody take a look at this, please, so I can merge it in? 😊

**Comment by lucasveil** [comment]
Scales perfectly for me @vnys ! Really good job!

File: package.yaml [hunk 5]
```diff
@@ -11,6 +11,7 @@ scripts:
   build:tokens: pnpm --filter @equinor/eds-tokens run build
   build:icons: pnpm --filter @equinor/eds-icons run build
   storybook: pnpm --filter ./libraries/core-react run storybook
+  graphics: pnpm --filter ./graphics/woman run dev
```
Comment (mimarz) [inline package.yaml:14]: Looks like a missed path change after moving to `assets`

**Review by mimarz — CHANGES_REQUESTED** [review]
Maybe the `woman` could be under `assets/graphics` instead of just `assets`. I sure we will get more graphics in the future :D

Apart from that it looks good 👍

**Comment by vnys** [comment]
> Maybe the `woman` could be under `assets/graphics` instead of just `assets`. I sure we will get more graphics in the future :D

I think we can change that later when we get more graphics, no need to do that now.

File: package.yaml (hunk 5 above)
Comment (vnys) [reply package.yaml:14]: Right, missed that – thanks 😊

Comment (vnys) [reply package.yaml:14]:
```suggestion
  graphics: pnpm --filter ./assets/woman run dev
```

Rename graphics → assets

**Review by mimarz — APPROVED** [review]

**Pull request merged**